src/jest.js

```diff
@@ -0,0 +1,30 @@
+// @ts-check
```

Comment:
We could also have a look at [`jest/recommended`](https://github.com/jest-community/eslint-plugin-jest?tab=readme-ov-file#recommended).